Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst

```diff
@@ -6,29 +6,111 @@
 ES6 in the TYPO3 Backend
 ========================
 
-..  todo: Add information on ES6 and its registration
-    https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/5
-
-..  seealso::
-    This chapter is still work in progress. For now see the changelog:
+..  versionchanged:: 12.0
+    Starting with TYPO3 v12.0 JavaScript ES6 modules may be used instead of
+    AMD modules, both in backend and frontend context.
+
+JavaScript node-js style path resolutions are managed by
+`importmaps <https://wicg.github.io/import-maps/>`_, which allow web
```

Comment:
The URL is a redirect to https://html.spec.whatwg.org/multipage/webappapis.html#import-maps